compute_tools/src/http/api.rs

```diff
@@ -49,6 +50,11 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::GET, "/num_cpus") => {
+            info!("serving /num_cpus GET request");
+            Response::new(Body::from(format!("{{\"num_cpus\":{}}}", num_cpus::get())))
```

Comment:
I would feel itchy if I produced JSON with string formatting; however, if you don't feel itchy about this, it's fine with me.

Reply:
will refactor one more thing then. There is a better approach.